live-ingester/src/integration-test/resources/expected-hxinsight-requests/create-document-request.yml

```diff
@@ -0,0 +1,39 @@
+url: /v1/ingestion-events
```

Comment:
The idea here is just to extract the static json from e.g. `CreateRequestIntegrationTest` and put it in a file where two tests can access it. I think what you're suggesting is to combine the tests so that we validate the output from the Connector directly against the remote schema, but I'm concerned that if the test starts failing then we won't know if it's due to our code changes or changes in the remote schema.

Reply:
Additionally I think we want to validate the exact output of the Connector (rather than just its validity), because otherwise we would pass tests by e.g. only producing delete requests.

Reply:
Ok, I get your point - the idea is to perform validation in two steps:
1. verify if connector output meats "our requirements",
2. verify if "our requirements" comply with OpenAPI spec
One downside what I see here is: maintain complexity - if someone will forget to update this static yaml request we might end up in situation where Connector will be sending not valid requests and we might notice it quite late

Reply:
Yes - I agree. It's important that all static requests are also included in e2e tests so that we know they can be produced by the application. I can't think of a neat way to verify that.

Reply:
I made a suggestion of how we could ensure we test all requests in the folder against the schema here: https://github.com/Alfresco/hxinsight-connector/pull/767#discussion_r1891664332

Reply:
I think, that we could leave `CreateRequestIntegrationTest` and others `usecase.e2e` tests as they are and introduce new test verifying if our code complies with OpenAPI spec. If only this new one test would fail, it would mean, that the spec has changed. While when more tests will start to fail, then someone messed something up in the code.
Our Endpoint-to-Endpoint tests are very handy and bring a lot of value presenting in a simple way what will be the outcome for different repo events, so we shouldn't change them.